packages/@internationalized/date/src/CalendarDate.ts

```diff
@@ -42,6 +42,7 @@ export class CalendarDate {
   // This prevents TypeScript from allowing other types with the same fields to match.
   // i.e. a ZonedDateTime should not be be passable to a parameter that expects CalendarDate.
   // If that behavior is desired, use the AnyCalendarDate interface instead.
+  // @ts-ignore
```

Comment:
This ignores the "unused variable" error which comes up due to the tsconfig for the docs checker.